health/HealthService.go

```diff
@@ -0,0 +1,33 @@
+package health
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// Service implements the service health endpoint.
+type Service struct {
+	isTestMode bool
+}
+
+// NewHealthService returns a new HealthService instance.
+func NewHealthService(isTestMode bool) *Service {
+	r := new(Service)
+	r.isTestMode = isTestMode
+	return r
+}
+
+// getHealthInfo returns the health info.
+func (s *Service) getHealthInfo() map[string]bool {
```

Comment:
HTTP health check in k8s is based on the http response code. Not on the response body.
Any response between 200 and 400 will be considered as success. See https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-http-request

We can encode some meaningful content as an extra information (for debugging, tests, or whatever reason) but we still have to return a proper response code.